templates/kubeconfig.tpl

```diff
@@ -6,17 +6,18 @@ clusters:
 - cluster:
     server: ${endpoint}
     certificate-authority-data: ${cluster_auth_base64}
-  name: ${cluster_name}
+  name: ${kubeconfig_name}
 
 contexts:
 - context:
-    cluster: ${cluster_name}
-    user: ${user_name}
-  name: ${context_name}
-current-context: ${context_name}
+    cluster: ${kubeconfig_name}
+    user: ${kubeconfig_name}
```

Comment:
Just wanting one more sanity check that this seems reasonable as a value to `user` to those involved here. I feel like this is the only place where `kubeconfig_name` might not make the most sense but I could very well stand corrected.

Reply:
With EKS, since it uses the heptio-authenticator, this field is irrelevant. It serves only as a string to match from the `context` to the `user`. It could be anything at all, as long as it matches. The only place where you see it is in the `AUTHINFO` column in the output of `kubectl config get-contexts`.

I think customising the single field just for cosmetic reasons in the output of `kubectl config get-contexts` is quite edge case. But perhaps @sdavids13 has more info for a different reason that I am unaware of?

Reply:
Got it. That's the exact feedback I was hoping to see: this field is more cosmetic than anything and the user doesn't actually align with a meaningful username anywhere else.

Reply:
@max-rocket-internet Yea, shouldn't need the ability to specify an alternative user -- just needs to be unique across all configuration files which is being done here. 👍